### PR TITLE
adds some fixes

### PR DESCRIPTION
**Pull request by MariMax**

### CODE REVIEW

File: projects/next-checkbox/src/lib/next-checkbox.component.ts [hunk 1]
```diff
@@ -18,22 +21,18 @@ let nextUniqueId = 0;
...
   @Input() tabIndex: number;
-
-  private _uniqueId = this.idGenerator();
-  @Input() id: string;
+  @Input() id = `next-checkbox-${++nextUniqueId}-input`;
 
   get inputId(): string {
-    if (!this.id) {
-      this.id = this._uniqueId;
-    }
```
Comment (NikaOrl) [inline projects/next-checkbox/src/lib/next-checkbox.component.ts:36]: This code was added in case of id=null or undefined (the attribute [id] was set but the variable equals to null or undefined). Without this code 4 tests fail.

**Review by mariia-zu — APPROVED** [review]

File: projects/next-checkbox/src/lib/next-checkbox.component.ts (hunk 1 above)
Comment (MariMax) [reply projects/next-checkbox/src/lib/next-checkbox.component.ts:36]: need to fix tests then, if user pass null or undefined, then it should set null or undefined, in user didn't pass anything it will be set to default value: `next-checkbox-${++nextUniqueId}-input` so I don't think you need this validation

Comment (NikaOrl) [reply projects/next-checkbox/src/lib/next-checkbox.component.ts:36]: Ok, then I'll fix tests. I just thought that the checkbox should generate id in case of some errors that set id to null or undefined.
